community/decorators.py

```diff
@@ -0,0 +1,31 @@
+LOGGING_CONFIG = {
```

Comment:
Hey, Django already setup logging for us, Please don't create a new one, just use the one Django already setup.
https://docs.djangoproject.com/en/2.0/topics/logging/

Reply:
If you need to add more handlers, see https://docs.djangoproject.com/en/2.0/topics/logging/#configuring-logging,
Do it inside settings.py